tools/vsimporter/src/vswriter/VSSolution.cpp

```diff
@@ -228,11 +228,16 @@ void VSSolution::write(std::ostream& out) const {
     if (m_version == 12) {
         out << "# Visual Studio 2013" << std::endl;
         out << "VisualStudioVersion = 12.0.21005.1" << std::endl;
-    } else {
+        out << "MinimumVisualStudioVersion = 10.0.40219.1" << std::endl;
+    } else if (m_version == 14) {
         out << "# Visual Studio 14" << std::endl;
         out << "VisualStudioVersion = 14.0.22823.1" << std::endl;
+        out << "MinimumVisualStudioVersion = 14.0.22823.1" << std::endl;
+    } else {
+        out << "# Visual Studio 15" << std::endl;
+        out << "VisualStudioVersion = 15.0.26228.4" << std::endl;
```

Comment:
Just picked 15.0.26228.4 and 15.0.26206.0 based on what I've seen but VS doesn't really care about VisualStudioVersion and MinimumVisualStudioVersion anyway. It'll update these two numbers as it pleases.

It really only cares about "# Visual Studio 15". That's where it's deciding compatibility for 2015 vs 2017.